Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-RUN if [ $(cat /etc/timezone) = "Asia/Shanghai" ] ; then go env -w GOPROXY=https://goproxy.cn,direct; fi
```

Comment:
can use `Arg` when you build, Do not use `go env -w GOPROXY` , it will cause CI's failure.

Reply:
next pr to solve